packages/aggregator/src/aggregationTypes.js

```diff
@@ -8,6 +8,7 @@ export const AGGREGATION_TYPES = {
   MOST_RECENT: 'MOST_RECENT',
   MOST_RECENT_PER_ORG_GROUP: 'MOST_RECENT_PER_ORG_GROUP',
   SUM: 'SUM',
+  SUM_UNTIL_THIS_DAY: 'SUM_UNTIL_THIS_DAY',
```

Comment:
Maybe `SUM_UNTIL_CURRENT_DAY`? "This" makes me think that we can specify which day, whereas it's always the current day.

Reply:
Also, I don't believe that the `UNTIL` represents what is going on at the moment. If there is data AFTER today's date, that data will still be included in the sum, so perhaps a rename here as well?